doc/ROS2/setup/Agent.md

```diff
@@ -2,6 +2,11 @@
 
 Sources: [here](https://micro-xrce-dds.docs.eprosima.com/en/latest/index.html)
 
+- [Installation](#installation)
+- [Using the agent with the serial interface](#using-the-agent-with-the-serial-interface)
+- [Using the agent with the wifi interface](#using-the-agent-with-the-wifi-interface)
+- [Testing the node](#testing-the-node)
+
 ## Installation
 
 Follow the instructions detailed [here](https://micro-xrce-dds.docs.eprosima.com/en/latest/installation.html). It has only been tested as standalone executable (without Docker neither using Snap).
```

Comment:
Please replace "executable" with agent, as there are 2 different standalone setups (agent and client).

Reply:
But the whole file is related to the Agent

Reply:
Ok we can keep for now, and update dependend on the story regarding Micro-Ros native support, where we eventually need the client libraries as well.